Bugfix images hero

YanaMoss opened the pull request:
Костя, сделал замечание, что картинки для героя должны быть в формате PNG. Исправила.

YanaMoss committed:
bugfix-images-hero

Перезалила картинки для героя в формате PNG.

YanaMoss committed:
Update hero-desktop@2x.png

Baklaikin reviewed (changes requested):
выбери, пожалуйста group 57 на макете, чтобы картинка была с прозрачным кусочком в левой части

YanaMoss committed:
add single image for hero

Добавила по просьбе Кости общий вариант картинки героя. Раздельный вариант не удаляла на случай, если останется время на анимацию героя с использованием раздельных картинок.

Baklaikin reviewed (approved):
отлично
